test/gw/conversion_test.py

```diff
@@ -480,6 +482,24 @@ def test_generate_all_bbh_parameters(self):
             self.expected_bbh_keys,
         )
 
+    def test_generate_all_bbh_default_parameters(self):
```

Comment:
This should be removed after the functions in source.py are refactored.

Reply:
Removed